src/main/java/frc/robot/Robot.java

```diff
@@ -33,6 +35,8 @@ public class Robot extends LoggedRobot {
     private final Timer timer = new Timer();
     private RobotContainer robotContainer;
     private Command autonomousCommand;
+    private Vision vision;
```

Comment:
I won't place the `Vision` here if you're not going to instantiate it. It will just make your robot crash if someday you`ll use it

Reply:
removed